learning_assistant/api.py

```diff
@@ -217,6 +217,6 @@ def get_message_history(courserun_key, user, message_count):
 
     Returns a number of messages equal to the message_count value.
     """
-    message_history = LearningAssistantMessage.objects.filter(
-        course_id=courserun_key, user=user).order_by('-created')[:message_count]
+    message_history = list(LearningAssistantMessage.objects.filter(
+        course_id=courserun_key, user=user).order_by('-created')[:message_count])[::-1]
```

Comment:
You can get the same affect by doing order_by('created'). Right now you're still sorting twice.

Reply:
Discussed offline and the alternative would be less performant. I've added a comment with a justification on this.